enigmatry-entry-blueprint-app/Enigmatry.Entry.Blueprint.App.esproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.VisualStudio.JavaScript.Sdk/1.0.784122">
+  <PropertyGroup>
+    <StartupCommand>npm start</StartupCommand>
+    <JavaScriptTestFramework>Jest</JavaScriptTestFramework>
+    <!-- Allows the build (or compile) script located on package.json to run on Build -->
+    <ShouldRunBuildScript>false</ShouldRunBuildScript>
+    <!-- Folder where production build objects will be placed -->
+    <BuildOutputFolder>$(MSBuildProjectDirectory)\dist\</BuildOutputFolder>
+    <BuildCommand>npm run build -- --configuration production</BuildCommand>
```

Comment:
This will automatically be called when the API project builds, because of the project reference.

Reply:
We usually develop FE in VS Code and BE (API) in VS. This is convenient for CI/CD since it's gonna simplify steps needed for build, but will slow down API development? Can we say that this is for release builds only?

Reply:
The ShouldRunBuildScript parameter on line 6 is set to false, which prevents running build from Visual Studio. However, the command on line 9 is triggered when the dotnet publish command is used (for example, in a CI/CD scenario).